### ISSUE TITLE
PHPWord + MODX

### ISSUE THREAD

**Issue by alexbom** [PHPOffice/PHPWord#1422]
This is:

- [x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpword)

### Expected Behavior

$phpWord get global variable without errors like in Readme :)

### Current Behavior

![modx phpword](https://user-images.githubusercontent.com/1853593/42587313-fa01884c-8542-11e8-8568-1c0be9716e9f.png)

### Failure Information

Same behavior on phpword/bootstrap.php & phpword/vendor/autoload.php
File is loaded, but the error is comming:

> Fatal error: Uncaught Error: Class 'PhpOffice\PhpWord\PhpWord' not found in...

### How to Reproduce

First tried in snippet, then tried in console simple installation code from Readme:

```php
<?php
require_once 'phpword/vendor/autoload.php';

$phpWord = new \PhpOffice\PhpWord\PhpWord();
```

### Context

* PHP version: 7.0.21
* PHPWord version: 0.14
* MODX version: 2.5.4-pl

Breaking brain for 2 days, my friends! :) Please, help )

**Comment by troosan** [comment]
I'm guessing this is not a PHPWord issue but rather a modx/composer issue.

**Comment by troosan** [comment]
Indeed, not sure what the relative path should be when executing code through `eval(...)` though
